rfc/rfc002-authentication-token.md

```diff
@@ -129,26 +129,13 @@ EN:PractitionerLogin:v2 Undersigned gives permission to Nuts foundation to make
 
 The time layout used is constructed as: `Monday, 2 January 2006 15:04:05` The name of the service provider must match the name that has been registered for this service provider. The name of the care organisation must match the name within the proof that has been published by the service provider. The signature must be made by cryptographic means which can be connected to the user.
 
-## 6. Authentication Token Container
+## 6. Verifiable Presentation
 
-The Authentication Token MUST be signed by the user. See section 7 for more detail on supported means. In order to support several signing means, the signed token MUST be encapsulated by a Authentication Token Container.
+The Authentication Token MUST be signed by the user. See section 7 for more detail on supported means. In order to support several signing means, the signed token MUST be encapsulated by a Container. As a container specification we use the [W3C Verifiable Presentation specification 1.0](https://www.w3.org/TR/vc-data-model/).
 
-The container MUST be formatted as a JSON object with the following members.
+It is recommended to reuse existing specifications for Verifiable Presentations (VP). If no existing specification exists, the type of the VP MUST be prepended with `Nuts`. This indicates that the VP is specified within this RFC.
 
-The "type" uniquely identifies the means used for creating the token. The value MUST be a lower case string without spaces. The means identifier gives the validator a hint about the contents of the token.
-
-The "token" member MUST contain the base64 encoded _Authentication Token_. The contents of the token can be anything and must be specified by the means.
-
-Other members than the ones specified here SHOULD NOT be used and MUST be ignored.
-
-Example of a Authentication Token Container:
-
-```yaml
-{
-  "type": "irma",
-  "token": "ewoiQGNvbnRleHQiOiAiaHR...FJJd05QN1N1dlAwSi9NL05WNTFWWnZxQ3lPKzdlRHdKOD0iCiAgfQp9Cn0=0"
-}
-```
+This specification uses the JSON-LD format for VPs since the JWT format isn't usable for all type of proofs. See the [linked data proofs](https://www.w3.org/TR/vc-data-model/#linked-data-proofs) section for more detail.
```

Comment:
doesn't VC always use JSON-LD (instead of JWT)

Reply:
Yes but this is a VP, not a VC